Makefile

```diff
@@ -228,6 +228,10 @@ lint-metrics:
 	./hack/prom-metric-linter/metric_name_linter.sh --operator-name="kubevirt" --sub-operator-name="kubevirt" --metrics-file=metrics.json
 	rm metrics.json
 
+lint-monitoring:
+	go install github.com/kubevirt/monitoring/monitoringlinter/cmd/monitoringlinter@latest
```

Comment:
You should probably add this to the builder image.

Reply:
I mean here: https://github.com/kubevirt/kubevirt/blob/9d36c93dc57f165807e1d339d7dffd8f5fd82231/hack/builder/Dockerfile

Reply:
Don't think this is possible until we complete the refactoring (also adding `make lint-monitoring` to the CI), right?

Reply:
I do not think they are related.
You can push it into the builder in a separate PR, I am not sure if the maintainers will allow you to add this `go install` here.

Reply:
Done, and now we can see the linter errors on https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/11454/pull-kubevirt-code-lint/1765752677913858048.
@EdDev can you PTAL again and let me know if anything else is missing from your POV?

Reply:
Very nice, thanks.

So you plan now to add the fixes to this PR as well? (to pass the linter failures)

Reply:
Thanks, it will be completed by https://github.com/kubevirt/kubevirt/pull/11184 and https://issues.redhat.com/browse/CNV-39013. This PR will be on hold until then.